cmd/cmd.go

```diff
@@ -800,9 +800,9 @@ func ShowHandler(cmd *cobra.Command, args []string) error {
 		case "parameters":
 			fmt.Println(resp.Parameters)
 		case "system":
-			fmt.Println(resp.System)
+			fmt.Print(resp.System)
```

Comment:
I think we still want to append a newline if
1. The content doesn't end in a newline and
2. We printing to stdout

Reply:
I can see your point - output minus a trailing newline is unaesthetic. But users benchmark against this as truth, when it's inexact. Perhaps we could mark truncated lines with '\no newline' and add -raw for cases needing precision?

Reply:
Also, I'm not seeing the same `cat` behavior your mentioning:

![CleanShot 2024-11-09 at 00 19 47@2x](https://github.com/user-attachments/assets/624ec19c-1c83-450d-9619-2d9f30ade122)

Reply:
I'm seeing this in Kitty and Terminal.app for both zsh and bash, on macOS:

```
# zsh
; echo -n x > nonewline.txt
; cat nonewline.txt 
x%
```

```
# bash
bash-3.2$ cat nonewline.txt 
xbash-3.2$ 
```

Reply:
Ah it was my editor `vi` adding a newline, not `cat`, sorry. This makes sense then, plus as you mentioned terminals add the `%` to be explicit:

<img width="340" alt="Screenshot 2024-11-09 at 12 40 52 PM" src="https://github.com/user-attachments/assets/ffdf4a19-379b-40d5-ac0d-e60b43a33bc0">